### ISSUE TITLE
Issue: Node.js: Configure logs in context

### ISSUE THREAD

**Issue by destumme** [newrelic/docs-website#4224]
### Tell us what you need

The portions in this page which talk about a log line simply needs the `trace.id` and `span.id` are misleading for those attempting to manually instrument log context in node without the winston enrichment.

I am attempting to configure logs with several fastify apis which requires manual instrumentation. Simply ensuring that the `span.id` and `trace.id` is logged seems to be insufficient. I was attempting to use the `getTraceMetadata` method to supply this information, which has just the trace and span ids. After looking over the implementation of the winston enrichment library, it became clear I should be using the `getLinkingMetadata` method when supplying this information to my logger.

It would be beneficial to have the documentation include some information for how to manually instrument the log context to include the `entity.guid` and other related fields returned from `getLinkingMetadata` that seems to be required for the Logs UI portion of the APM page to correctly show associated log statements.

### Anything else you'd like to share?

Add other context like screenshots, links to other docs, and information about
your environment (operating system, application framework, etc.).

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/logs/logs-context/configure-logs-context-nodejs/
* **OS**: Mac OS 10.15.7
* **Browser**: Chrome 94.0.4606.71
* **Device**: Unknown

**Comment by destumme** [comment]
Appreciate the quick reply. 

If it's helpful at all, for my api's I'm using pino logger, which has a mixin() function on the configuration, https://github.com/pinojs/pino/blob/master/docs/api.md#mixin-function.

Once I realized I needed to use the linkingMetatdata, simply returning the getLinkingMetadata from that function was all I needed to do. The 'Logs' tab in APM was populated as I expected after I set that up.
